### DISCUSSION TITLE
Is there a metric or a way to monitor otlpreceiver gRPC authenticator errors?

### DISCUSSION THREAD

**Discussion by DWonMtl** [open-telemetry/opentelemetry-collector#5442] [Q&A]
Hi!

We are currently doing some small tests on our opentelemetry collector to see how the it behave when we have invalid gRPC authentication.

We uses a proprietary extension to perform our authentication and when the authentication fails, an error is returned. We can see this error in our tracing client as expected. However, from the opentelemetry metrics point of view, these calls are totally absent. 

Our extension doesn't generate any metrics.

Is there a metric, or any ways, to access the otlp receiver gRPC number authenticator errors? 

If not, is there a plan to add such metrics?

Otherwise, should/can we add our own metrics to our authenticator extension?

**Comment by jpkrohling** [comment]
No, but this is a great candidate once #4198 is done.
